### PR TITLE
Back-end for form

### PR DESCRIPTION
**Pull request by KKA-0**

### CODE REVIEW

**Comment by KKA-0** [comment]
Ouff

**Comment by jadonharsh109** [comment]
Thanks Buddy

File: insert.php [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+		
+		// Check connection
+		if($conn === false){
+			die("ERROR: Could not connect. "
+				. mysqli_connect_error());
+		}
+		
+		// Taking all 5 values from the form data(input)
+       
+		$name = $_REQUEST['name'];
```
Comment (KKA-0) [inline insert.php:26]: making post method to variables

File: insert.php [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+		$email = $_REQUEST['email'];
+		$number = $_REQUEST['number'];
+        $message = $_REQUEST['message'];
+		
+		// Performing insert query execution
+		// here our table name is college
+		$sql = "INSERT INTO contact SET
```
Comment (KKA-0) [inline insert.php:33]: contacting with db and inserting the values

**Pull request merged**